src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -669,8 +593,11 @@ export class ResultsViewPageStore {
     }
 
     @computed
-    public get excludeGermlineMutations() {
-        return this.urlWrapper.query.exclude_germline_mutations === 'true';
+    public get includeGermlineMutations() {
+        return (
+            !!this.urlWrapper.query.exclude_germline_mutations ||
```

Comment:
Well, the filter menu in the ResultsView at present has the ability to exclude VUS and germline. The new filter menu that we use in study view and comparison view have more sophisticated options to include options. Since we created a common class _DriverAnnotationSettings_ shared by ResultsView, StudyView and ComparisonView an agreement on the interface had to be made. Since there were more options/members in the StudyView and ComparisonView I changed the interface to _include..._ everywhere. The change of this interface made it necesary to update method names in _ResultsViewPageStore.ts_.

Reply:
okay, that makes sense